scripts/check-docker-config.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# zenoss-inspector-info
+# zenoss-inspector-tags docker config verify
+# zenoss-inspector-deps docker-config.sh
+
+grep -E -i '--exec-opt native.cgroupdriver=cgroupfs' docker-config.sh.stdout &>/dev/null
```

Comment:
This check is wrong somehow. I ran it on pj-cc1 (10.88.111.102), and got the following:
```
check-docker-config.sh
Incorrect docker config: Add '--exec-opt native.cgroupdriver=cgroupfs' to /etc/sysconfig/docker
```
But here are the contents of /etc/sysconfig/docker:
```
OPTIONS="-s devicemapper --exec-opt native.cgroupdriver=cgroupfs --storage-opt dm.thinpooldev=/dev/mapper/docker-docker--pool --dns=172.17.42.1 --bip=172.17.42.1/16"
```

I suspect the problem is that grep sees the "--exec-opt" as a CLI option to grep, rather than a string to search for.  Also, since you're not specifying a regular expression, the `-E` is not necessary.